contrib/test/crio-integration-playbook.yaml

```diff
@@ -8,53 +8,21 @@
     cni_commit: dcf7368eeab15e2affc6256f0bb1e84dd46a34de
   tasks:
   - name: Update all packages
-    yum:
+    package:
       name: '*'
       state: latest
     async: 600
     poll: 10
     when: (ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS')
     ignore_errors: true
-  - name: Update all packages on Fedora
-    dnf:
+  - name: Update all packages
+    package:
       name: '*'
       state: latest
     async: 600
     poll: 10
-    when: ansible_distribution == 'Fedora'
-  - name: Make sure we have all required packages
-    yum:
-      name: "{{ item }}"
-      state: latest
-    with_items:
-      - wget
-      - git
-      - make
-      - gcc
-      - tar
-      - libseccomp-devel
-      - golang
-      - glib2-devel
-      - glibc-static
-      - container-selinux
-      - btrfs-progs-devel
-      - device-mapper-devel
-      - ostree-devel
-      - glibc-devel
-      - gpgme-devel
-      - libassuan-devel
-      - libgpg-error-devel
-      - pkgconfig
-      - skopeo-containers
-      - oci-systemd-hook
-      - oci-register-machine
-      - oci-umount
-      - socat
-    async: 600
-    poll: 10
-    when: (ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS')
-  - name: Make sure we have all required packages on Fedora
-    dnf:
+  - name: Make sure we have all required packages on Fedora and its downstreams
+    package:
```

Comment:
Don't we still need to give it a list?

Reply:
We should still be doing that, pairing each of the packages in the `with_items` list with `state: latest` using a [loop construct](http://docs.ansible.com/ansible/latest/playbooks_loops.html#standard-loops).  We were doing it before, but the combination of merging the two tasks while using a different task type for the result makes the diff look a little funky.